latest/ug/automode/auto-accelerated.adoc

```diff
@@ -0,0 +1,342 @@
+[.topic]
+[#auto-accelerated]
+= Deploy an accelerated workload
+:info_titleabbrev: Deploy accelerated workload
+
+include::../attributes.txt[]
+
+
+This tutorial demonstrates how Amazon EKS Auto Mode simplifies launching GPU-based workloads. Amazon EKS Auto Mode streamlines operations beyond the cluster itself by automating key infrastructure components providing compute, networking, load balancing, storage, and Identity Access and Management capabilities out of the box. 
+
+Amazon EKS Auto Mode includes the drivers required for certain instance types, such as NVIDIA and {aws} Neuron drivers. You do not have to install or update these drivers. 
+
+EKS Auto Mode manages the drivers for: 
+
+* link:ai/machine-learning/trainium/[{aws} Trainium, type="marketing"]
+* link:ai/machine-learning/inferentia/[{aws} Inferentia, type="marketing"]
+* NVIDIA GPU link:ec2/latest/instancetypes/ac.html[Amazon EC2 accelerated instances,type="documentation"]
+** Including the https://github.com/NVIDIA/k8s-device-plugin[NVIDIA device plugin for Kubernetes]. EKS Auto Mode includes this plugin, and is not visible as a daemon set within your cluster. 
+
+Amazon EKS Auto Mode also includes support for link:hpc/efa/[Elastic Fabric Adapter,type="marketing"].
+
+//GDC: What does the EFA thing mean?
+
+Amazon EKS Auto Mode eliminates the toil of GPU driver management. 
+
+You can also benefit from cost savings by scaling the cluster to zero. You can configure EKS Auto Mode to terminate instances when no workloads are running. This is useful for batch based inference workloads.
+
+The following provides an example of how to launch GPU-based workloads with Amazon EKS Auto Mode.
+
+== Prerequisites
+
+* A Kubernetes cluster with Amazon EKS Auto Mode configured.
+* A `default` EKS Node class as created when the `general-purpose` or `system` Managed Node Pools are enabled.
+
+== Step 1: Deploy a GPU workload
+
+In this example, you will create a NodePool for NVIDIA inference based workloads that requires 45GB GPU memory. With EKS Auto Mode, you use Kubernetes scheduling constraints to define your instance requirements. 
```

Comment:
the phrasing was a bit long, and theres nothing specific to inference
```suggestion
In this example, you will create a NodePool for NVIDIA based workloads that requires 45GB GPU memory. With EKS Auto Mode, you use Kubernetes scheduling constraints to define your instance requirements. 
```

Reply:
Suggested:

"In this example, you will create a NodePool for NVIDIA based workloads that requires `l40s` GPU due to GPU memory requirements. With EKS Auto Mode, you use Kubernetes scheduling constraints to define your instance requirements."